ci-scripts/firebase_publish_report.py

```diff
@@ -65,8 +65,12 @@ def lte_integ_test(args):
     # failing test cases, because of which CI report always shows lte integ
     # test as success. Here we read the CI status from file for more accurate
     # lte integ test execution status
-    with open('test_status.txt', 'r') as file:
-        verdict = file.read().rstrip()
+    if os.path.exists("test_status.txt"):
+        with open('test_status.txt', 'r') as file:
+            status_file_content = file.read().rstrip()
+            expected_verdict_list = ["pass", "fail"]
+            if status_file_content in expected_verdict_list:
+                verdict = status_file_content
```

Comment:
If we cannot find the test status should we report a "fail" verdict?

Reply:
I agree to this point to some extent that if test status is not available that means it is failing. But I believe when we are saying status of lte integ test, then it should be sanity test suite execution status and not the environment setup status. So if it fails during environment setup, this is CI setup issue (because sanity suite may still be passing, but could not execute because of ci environment issues) and the integ test status for that ci run is basically inconclusive. If it fails during the sanity test suite run, then it is basically the failure of lte integ test.

Reply:
Both is fine with me